### ISSUE TITLE
南京晓庄学院通知公告RSSHub docker compose部署版本路由失效

### ISSUE THREAD

**Issue by real-jiakai** [DIYgod/RSSHub#11245]
### 路由地址

```routes
/njxzc/tzgg
```

### 完整路由地址

```fullroutes
无
```

### 相关文档

https://docs.rsshub.app/university.html#nan-jing-xiao-zhuang-xue-yuan

### 预期是什么？

实现rsshub docker部署版本南京晓庄学院官网通知公告rss链接正常访问。
另外想请教一下，本地启的rsshub服务访问学校官网的rss链接正常，但是服务器部署的rsshub访问学校官网的rss链接不正常的原因。

### 实际发生了什么？

本地pull了rsshub仓库的最新代码以后，`npm run dev`，生成的南京晓庄学院官网通知公告rss链接正常。

![image](https://user-images.githubusercontent.com/82650452/200709148-b287e339-ef69-439c-97f2-2022f122b672.png)

![image](https://user-images.githubusercontent.com/82650452/200709264-e78c1792-ba3b-4935-825b-39e69abefe01.png)

但是无论是国外的服务器，还是国内的服务器，docker部署的rsshub均访问南京晓庄学院官网通知公告rss链接失败。

![image](https://user-images.githubusercontent.com/82650452/200709370-4f7dc755-c623-48ea-8176-1d861ee966df.png)

服务器采用curl测试学校通知公告网址，能获得完整的源码，里面包含了需要提取的标签。学校通知公告不是仅允许校园网访问的，是公开的。

![image](https://user-images.githubusercontent.com/82650452/200710695-b5210882-be1c-4364-b446-e6c7387aa8d6.png)

![image](https://user-images.githubusercontent.com/82650452/200711261-f56025ac-103a-466b-b338-e5a927f1d0a8.png)

### 部署

自建

### 部署相关信息

OS: Linux, Node: v16.18.0, Docker v 20.10.19, build d85ef84

### 额外信息

```shell
# 进入rsshub容器内部，获取的报错日志信息如下所示：
root@e2f88e91107f:/app/logs# cat error.log 
{"level":"error","message":"Error in /tzgg: Cannot read properties of null (reading 'replace')"}
{"level":"error","message":"Error in /tzgg: Cannot read properties of null (reading 'replace')"}
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

**Comment by TonyRL** [comment]
/test
```routes
/njxzc/tzgg
```

**Comment by TonyRL** [comment]
https://github.com/DIYgod/RSSHub/blob/68cf1f2ca037d44768832f6099ce260c07171b23/lib/v2/njxzc/home.js#L43-L44

**Comment by real-jiakai** [comment]
> https://github.com/DIYgod/RSSHub/blob/68cf1f2ca037d44768832f6099ce260c07171b23/lib/v2/njxzc/home.js#L43-L44

I should remove these two lines and PR? There is no need to replace the relative path with the absolute path?

![image](https://user-images.githubusercontent.com/82650452/200718586-ac4958fc-1f62-429d-b4b5-fecf07a43012.png)

**Comment by real-jiakai** [comment]
With these two lines. The running result:

![image](https://user-images.githubusercontent.com/82650452/200720260-d555590d-ebfc-4582-a5ae-ad87ce1122dd.png)

Without these two lines. The running result:

![image](https://user-images.githubusercontent.com/82650452/200720319-3bd7f7ed-5867-444b-8adc-eef329d03a58.png)

我应该怎么做才能使得这个rss回复正常，期待您的回复！

**Comment by TonyRL** [comment]
> https://github.com/DIYgod/RSSHub/blob/68cf1f2ca037d44768832f6099ce260c07171b23/lib/v2/njxzc/home.js#L43-L44

Some of the pages are being redirected as shown in https://github.com/DIYgod/RSSHub/actions/runs/3424668923/jobs/5704658765#step:8:26
In this case, `$('.wp_articlecontent')` does not exist and I don't think you can use `.replace()` on something does not exist

**Comment by real-jiakai** [comment]
> > https://github.com/DIYgod/RSSHub/blob/68cf1f2ca037d44768832f6099ce260c07171b23/lib/v2/njxzc/home.js#L43-L44
> 
> Some of the pages are being redirected as shown in https://github.com/DIYgod/RSSHub/actions/runs/3424668923/jobs/5704658765#step:8:26 In this case, `$('.wp_articlecontent')` does not exist and I don't think you can use `.replace()` on something does not exist

Ok, I will delete these two lines and resubmit PR. Thank you for your advice.

**Comment by TonyRL** [comment]
Closed in #11253
